hawc/apps/form_library/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from . import views
+
+app_name = "form_library"
+urlpatterns = [
+    # Create a Data Extraction form
+    path(
+        "form/create/",
+        views.CreateDataExtractionView.as_view(),
+        name="form_create",
+    ),
+]
```

Comment:
add `if settings.HAWC_FEATURES.ENABLE_DYNAMIC_FORMS else []` to the closing bracket to make this a ternary.